Pearson-Advance/frontend-app-institution-portal
# feat: Input for name and last name for instructor add modal

# Description

This PR adds inputs for instructor name and last name.

This PR resolves [PADV-1110](https://agile-jira.pearson.com/browse/PADV-1110)

## Change log
- Added right margin in cancel button.
- Refactor in imports order.
- Refactor in the internal logic to handle the component state logic.
- Fix minor bug when the last access is null.
     - When an instructor is created with all fields the endpoint return a "null" as result the table shows a incorrect date:
 
![F](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/75bc19b5-5c0b-43d7-9a8c-26dd3791b4b2)


- Fix, disabled button if the email is not provided.
- Renamed _test_ folder for __test__ ( convention is double "_" )


## Screenshots
### Initial
![1aa](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/c940fbff-765f-42c6-aefb-5ecf745d1985)

### Loading
![1alog](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/991e98fb-c7ed-4d66-aa77-cff45fe41e42)

### Error
![1aaerror](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/bbd99242-8ba5-4f0f-be39-4612ae1ce595)

### Success
![aagod](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/da1cb0e4-b71d-43a6-b843-ebad4210ed5c)



### How to test

Clone the Institution Portal MFE
```Bash
     git clone <url>
```
Install the packages 📦.
```Bash
     npm i
```
Start project.
```Bash
     npm start
```

Finally, go to:

- http://localhost:1980/instructors


and see the result.
